project2/index.html

```diff
@@ -0,0 +1,98 @@
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <link href="https://fonts.googleapis.com/css?family=Merriweather:400,700&display=swap" rel="stylesheet">
```

Comment:
You want to combine these stylesheet references on lines 7 and 8 into one call, like so:
```suggestion
    <link href="https://fonts.googleapis.com/css?family=Merriweather:400,700|Noto+Sans+JP:500,700&display=swap" rel="stylesheet">
```